src/commands/fizzbuzz.js

```diff
@@ -33,19 +33,34 @@ class FizzBuzz extends Aquarius.Command {
   }
 
   message(msg) {
-    const input = Aquarius.Triggers.messageTriggered(msg, /^fizzbuzz ([\d]+)$/i);
```

Comment:
Gotcha. I think a better way might be saying 

```
if (input) {
... 
} else if (msg.startsWith('.fizzbuzz') {
  sendMsg('please use this correctly')
}
```

Seems a little cleaner than me. Lemme think on it a bit though